### DISCUSSION TITLE
Trying to install ffmpeg on old Mac (for Audacity)

### DISCUSSION THREAD

**Discussion by rcuttill** [Homebrew/discussions#4885] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.18
ORIGIN: https://github.com/Homebrew/brew
HEAD: bf8991f0d777fe8e33105f0cd7bf1e5993eed60a
Last commit: 8 days ago
Core tap JSON: 06 Nov 06:10 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.10 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: quad-core 64-bit ivybridge
Clang: 12.0.0 build 1200
Git: 2.24.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 10.15.7-x86_64
CLT: 12.4.0.0.1.1610135815
Xcode: N/A
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You are using macOS 10.15.
We (and Apple) do not provide support for this old version.
It is expected behaviour that some formulae will fail to build in this old version.
It is expected behaviour that Homebrew will be buggy and slow.
Do not create any issues about this on Homebrew's GitHub repositories.
Do not create any issues even if you think this message is unrelated.
Any opened issues will be immediately closed without response.
Do not ask for help from Homebrew or its maintainers on social media.
You may ask for help in Homebrew's discussions but are unlikely to receive a response.
Try to figure out the problem yourself and submit a fix as a pull request.
We will review it but may or may not accept it.
```

### Description of issue

Can anyone tell me what's wrong. I wanted ffmpeg for Audacity. I tried installing FFmpeg the standard way and got FFmpeg library not found when looking at preferences/libraries even though it is clearly there. This is true both for Audacity 3.3.3 and 3.4.0. 

Found a suggestion that installing via homebrew works. So installed homebrew and got this -
"Already downloaded: /Users/raymondcuttill/Library/Caches/Homebrew/downloads/e163886f3d57674f022eb584f07ef6d71dbbf58fb39c481a2c516ce6bfd4af18--Example.ogg
Error: ffmpeg: Failed to download resource "libvorbis"
Download failed: Homebrew-installed `curl` is not installed for: https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz"

Don't really understand what the problem is or how to fix it.

**Comment by ZhongRuoyu** [comment]
That means Homebrew-provided `curl` is needed to download `libvorbis`, an `ffmpeg` dependency. Please run `brew install curl` and try again.

**Reply by rcuttill** [reply]
Thanks for your help. I've found another way round. On
https://support.buanzo.org/t/old-macbook-pro-ffmpeg-issues/37/3
is says
"Hi, the .dmg package for audacity > 3.2.0 was not working. We confirmed with a few users. I have updated the site, please use the link RECOMMENDED download: [ffmpeg_64bit_osx.pkg 6](https://lame.buanzo.org/ffmpeg_64bit_osx.pkg)"
That now works. I may update curl later. Thanks again.

**Reply by zdrive1** [reply]
where can i get my hands on Audacity i also have a mac on 10.15 ..thanks

**Comment by M1111991** [comment]
> ### مخرج ل`brew config`
> ```shell
> HOMEBREW_VERSION: 4.1.18
> ORIGIN: https://github.com/Homebrew/brew
> HEAD: bf8991f0d777fe8e33105f0cd7bf1e5993eed60a
> Last commit: 8 days ago
> Core tap JSON: 06 Nov 06:10 UTC
> HOMEBREW_PREFIX: /usr/local
> HOMEBREW_CASK_OPTS: []
> HOMEBREW_MAKE_JOBS: 4
> Homebrew Ruby: 2.6.10 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
> CPU: quad-core 64-bit ivybridge
> Clang: 12.0.0 build 1200
> Git: 2.24.3 => /Library/Developer/CommandLineTools/usr/bin/git
> Curl: 7.64.1 => /usr/bin/curl
> macOS: 10.15.7-x86_64
> CLT: 12.4.0.0.1.1610135815
> Xcode: N/A
> ```
> 
> ### مخرج ل`brew doctor`
> ```shell
> Please note that these warnings are just used to help the Homebrew maintainers
> with debugging if you file an issue. If everything you use Homebrew for is
> working fine: please don't worry or file an issue; just ignore this. Thanks!
> 
> Warning: You are using macOS 10.15.
> We (and Apple) do not provide support for this old version.
> It is expected behaviour that some formulae will fail to build in this old version.
> It is expected behaviour that Homebrew will be buggy and slow.
> Do not create any issues about this on Homebrew's GitHub repositories.
> Do not create any issues even if you think this message is unrelated.
> Any opened issues will be immediately closed without response.
> Do not ask for help from Homebrew or its maintainers on social media.
> You may ask for help in Homebrew's discussions but are unlikely to receive a response.
> Try to figure out the problem yourself and submit a fix as a pull request.
> We will review it but may or may not accept it.
> ```
> 
> ### وصف المشكلة
> يمكن لأحد أن يقول لي ما هو الخطأ. أردت ffmpeg للجرأة. لقد حاولت تثبيت FFmpeg بالطريقة القياسية ولم يتم العثور على مكتبة FFmpeg عند النظر إلى التفضيلات/المكتبات على الرغم من وجودها بوضوح. وينطبق هذا على كل من Audacity 3.3.3 و3.4.0.
> 
> وجدت اقتراحًا بأن التثبيت عبر البيرة المنزلية يعمل. لذلك قمت بتثبيت برنامج homebrew وحصلت على هذا - "تم التنزيل بالفعل: /Users/raymondcuttill/Library/Caches/Homebrew/downloads/e163886f3d57674f022eb584f07ef6d71dbbf58fb39c481a2c516ce6bfd4af18--Example.ogg Error: ffmpeg: Failed to download المورد " libvorbis" فشل التنزيل : لم يتم تثبيت Homebrew المثبت `curl`لـ: https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz "
> 
> لا أفهم حقًا ما هي المشكلة أو كيفية حلها.
